### ISSUE TITLE
Admin application fails on Python 3.10 in version 2.23.0

### ISSUE THREAD

**Issue by kszys** [web2py/web2py#2454]
**Describe the bug**

When opening the list of errors in a custom application using the `admin` application on a freshly downloaded web2py version 2.23.0 (from 27th December 2022), it fails with an error when run on Pyhon 3.10 with the error: 

```
<class 'TypeError'> a bytes-like object is required, not 'str'
```

**To Reproduce**

1. Make sure you have some errors in `<your app>`
2. Go to `/admin/errors/<your app>` on web2py running on Python 3.10

**Expected behavior**

A list of errors in `<your app>` should be shown.

**Screenshots**

<img width="784" alt="Screenshot 2023-01-03 at 12 52 40" src="https://user-images.githubusercontent.com/7057374/210352650-9f54d852-ee49-4185-aa72-e5acd9064ccf.png">

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version 108

**Comment by leonelcamara** [comment]
Are these old errors possibly created with another python version?

**Comment by kszys** [comment]
No, I have not changed the python version, when testing, so it is unlikely. Anyway, I solved it by adding more `except` statements. It already excepts `IOError` and `EOFError`, I just added to an `except Exception` to get rid of any other things that can go wrong. Since ignoring other errors does not seem to be a problem, ignoring this one either.

**Comment by leonelcamara** [comment]
This was fixed
